cypress/cypress.config.ts

```diff
@@ -0,0 +1,12 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  e2e: {
+    retries: 3,
+    supportFile: 'support/e2e.ts',
+    specPattern: 'e2e/**/*.cy.ts',
+    fixturesFolder: 'fixtures/',
+    video: false,
+    screenshotOnRunFailure: false,
+  },
+});
```

Comment:
Can we update to use import - 
```suggestion
import { defineConfig } from 'cypress'

export default defineConfig({
  e2e: {
    retries: 3,
    supportFile: 'support/e2e.ts',
    specPattern: 'e2e/**/*.cy.ts',
    fixturesFolder: 'fixtures/',
    video: false,
    screenshotOnRunFailure: false,
  },
});

```